### ISSUE TITLE
Inconsistent src requirements

### ISSUE THREAD

**Issue by mudlabs** [bradmartin/nativescript-lottie#37]
There's an inconsistency with how the `src` must be described in `xml`.
```xml
<!-- Works on iOS, but not Android -->
<Lottie:LottieView src="lottie" />

<!-- Works on both platforms -->
<Lottie:LottieView src="lottie.json" />
```
Given all Lottie assets are `JSON` files it seems redundant having to include the extension.

**Comment by itstheceo** [comment]
This is a pretty good starter task for someone with the time! Though the workaround would be to just specify the file extension for now

**Comment by bradmartin** [comment]
Closed via #46
